db/hints/manager.cc

```diff
@@ -873,11 +873,11 @@ future<> manager::perform_migration() {
     const auto lock = co_await seastar::get_units(_drain_lock, 1);
 
     using lock_type = std::unique_lock<seastar::shared_mutex>;
-    // We're taking this lock because we're about to stop endpoint managers here, wheras
+    // We're taking this lock because we're about to stop endpoint managers here, whereas
```

Comment:
Spelling fixes are no longer needed, they were fixed by https://github.com/scylladb/scylladb/issues/18567 which is already on master